asyncapi/json-schemas/operational/assignment/attempt/request/assignment-attempt-request.md

```diff
@@ -106,25 +106,35 @@ The DatedServiceJourneyId is `RUT:DatedServiceJourney:068153825c58c7d3e26a53ae23
 - If the attempt request succeeds, the vehicle will be assigned the new plan `AssignmentState.assigned=true`
 - If the attempt request fails, the state of the vehicle is `AssignmentState.assigned=false`
   - The reason for failing will be available under the [response topic](../response/assignment-attempt-response.md)
-- Contents of a signOn-attempt can now be based on vehicleTask, or a provided list of DatedServiceJourneys. 
-  - If a list of DatedServiceJourneys is provided, the provided vehicleTask is not included in the attempt 
-  - VehicleTask:
+- Contents of a signOn-attempt can now be based on vehicleTask, or a provided list of either DatedServiceJourneys or Journeys. If a list of DatedServiceJourneys or Journeys is provided, the provided vehicleTask is not included in the attempt
+- Options for sign on:
+  - Deprecated `VehicleTask`:
+    - Please use a list of either `DatedServiceJourneys` or `Journeys`
     - Require the fields `vehicleTaskId` and `serviceWindow`.
       - `vehicleTaskId`: Can be found in the common file in the NeTEx export under this path `VehicleScheduleFrame.blocks[].Block.PrivateCode`
       - `serviceWindow`: Defines a time range for which journeys the vehicle should be signed on.
         The vehicle will be signed on to all the journeys in the vehicle task between provided times.
         Signed on journeys may be part of 1 or more `Block`s.
         Times provided may belong to the same calendar date, or 2 consecutive dates.
         Attempts not containing valid date times for `firstDepartureDateTime` and `lastArrivalDateTimes` will be rejected, resulting in `AssignmentState.assigned=false`
-  - A list of DatedServiceJourneys:
+  - A list of `DatedServiceJourneys`:
     - `datedServiceJourneyId`: Can be found in the respective Journey file in the NeTEx export (See above example xml)
     - `serviceWindow`: Optional: Defines a time range for which calls in the journey the vehicle should be signed on. If not provided, the entire journey is included 
+  - A list of `Journeys`:
+    - `vehicleJourneyId`: Required if calls are not provided. Also known as 'turnummer'/trip number of the journey.
+    - `lineId`: Required if calls are not provided. Id for the line, e.g.: 'RUT:Line:32'. RUT:Line:0 can be used for DeadRuns
+    - `departureDateTime`: Departure date time for the first call in the journey. This field is used to pinpoint the exact dated journey to be serviced
```

Comment:
planned

Reply:
Hvilken tenker du, @larshoidahl ?
1. Departure date time for the first planned call in the journey
2. Departure date time for the first call in the planned journey
3. Departure date time for the first planned call in the planned journey